packages/evolution-generator/src/scripts/generate_questionnaire_dictionary.py

```
@@ -10,6 +10,19 @@
 from helpers.generator_helpers import get_data_from_excel, clean_text
 
 
+def parse_int_cell_value(value):
+    """Parse an Excel cell value as an integer (handles numeric strings)."""
+    if value is None or value == "":
+        return None
+    if isinstance(value, bool):
+        raise TypeError(f"Expected a numeric cell value, got boolean: {value}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)